src/account/UpgradeableModularAccount.sol

```diff
@@ -137,7 +138,7 @@ contract UpgradeableModularAccount is
 
         PostExecToRun[] memory postExecHooks;
         // Cache post-exec hooks in memory
-        postExecHooks = _doPreExecHooks(msg.sig, msg.data);
+        postExecHooks = _doPreExecHooks(getAccountStorage().selectorData[msg.sig].executionHooks, msg.data);
```

Comment:
Should `permissionHook` run here?

Reply:
Hmm not sure how to do that - permission hooks are associated with a validation function, if a call enters the fallback it wouldn't contain validation information so we wouldn't know which set of permission hooks to run